### ISSUE TITLE
Screen reader issue when no hash in URL

### ISSUE THREAD

**Issue by TristanLavalou** [rafgraph/react-router-hash-link#104]
Hello,

I'm facing an accessibility issue. When using Chrome's Screen Reader ChromeVox, a link is read twice when it's pointing on an URL which doesn't have an hash / anchor in it.

For example, using your online demo, if you're moving to "Section One" using the keyboard, entering it, then entering "Home", the link will be read twice and the screen reader focus will stay on it, even if the link was removed of the DOM.
